### ISSUE TITLE
JSON.stringify success message

### ISSUE THREAD

**Issue by arabold** [ashiina/lambda-local#6]
Just a small improvement request: If you pass in an object to `context.succeed()`, the content is dumped using `console.log()` which will not render nested objects properly. It would be better to use `JSON.stringify`. I have modified my local `lambda-local` as follows:

```javascript
Context.done = function (err, message) {
        console.log("END");
        console.log("\n");
        console.log("Message");
        console.log("------");
        console.log(typeof message === "object" ?
                JSON.stringify(message, null, "\t") : message);

        process.exit();
};
```

**Comment by ashiina** [comment]
thanks for this! will add this for the next version

**Comment by ashiina** [comment]
This fix is not included in 0.0.7! please update
